components/transactions/TransactionDetails.js

```diff
@@ -120,6 +121,7 @@ const TransactionDetails = ({
           {displayActions && ( // Let us overide so we can hide buttons in the collective page
             <React.Fragment>
               {permissions?.canRefund && !isRefunded && <TransactionRefundButton id={id} />}
+              {permissions?.canReject && !isRefunded && <TransactionRejectButton id={id} />}
```

Comment:
We can eventually keep that for a future iteration, but to me it seems ok to reject a refunded contribution. For example, if the host (or you) refund the contribution, but then you still want to remove the member.

Reply:
That's a really good point! I will change that.

Reply:
Making a note for myself: this means that we need to put logic in the API for checking whether the transaction has already been refunded or not so we don't try to do it again as part of the 'reject' flow.